jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListPresenter.java

```diff
@@ -323,7 +322,15 @@ public void callback(Void nothing) {
         taskSelected.fire( new TaskSelectionEvent( selectedServerTemplate, task.getDeploymentId(),task.getTaskId(), task.getTaskName() ) );
     }
 
-    public abstract Menus getMenus();
+    @WorkbenchMenu
+    public Menus getMenus(){
```

Comment:
It seems that the `@WorkbenchMenu` annotation only works in subclasses. I found that a similar issue may be happening in `AssetManagementPerspective`